sw/device/tests/aon_timer_irq_test.c

```diff
@@ -173,17 +186,21 @@ bool test_main(void) {
       mmio_region_from_addr(TOP_EARLGREY_RV_PLIC_BASE_ADDR);
   CHECK_DIF_OK(dif_rv_plic_init(plic_base_addr, &plic));
 
-  // Enable all the AON interrupts to check if the timer will fire only the
-  // correct one.
+  // Enable all the AON interrupts used in this test.
   rv_plic_testutils_irq_range_enable(
       &plic, kPlicTarget, kTopEarlgreyPlicIrqIdAonTimerAonWkupTimerExpired,
       kTopEarlgreyPlicIrqIdAonTimerAonWdogTimerBark);
 
   // Executing the test using a randon time between 1 and 15 ms to make sure
   // the aon timer is generating the interrupt after the choosen time and theres
   // no error in the reference time measurement.
-  uint32_t sleep_time = rand_testutils_gen32_range(1, 15) * 1000;
-  execute_test(&aon_timer, sleep_time);
+  uint64_t irq_time = rand_testutils_gen32_range(1, 15) * 1000;
```

Comment:
You could drop a factor of 100 in this multiplication: setting a very large timeout is not that useful, and 15 minutes for such a test to run is excessive.

Reply:
Hi @matutem, the timeout used is between 1 and 15 **milliseconds**. Below milliseconds I started to see some strange behaviors on Verilator (I guess it is because of the time taken to log messages).